.editorconfig

```diff
@@ -1,2 +1,2 @@
-[*.{ql,qll,qlref,dbscheme,qhelp,js,html}]
+[*.{ql,qll,qlref,dbscheme,qhelp,html,js,ts,json,yml}]

```

Comment:
(the change for `.json` and `.yml` should be in the previous commit)

Reply:
You're right, of course, but I don't think it's worth going back to fix at this point.